geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -85,7 +97,20 @@ case class CoprocessorPlan(sft: SimpleFeatureType,
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = {
     import org.locationtech.geomesa.index.conf.QueryHints.RichHints
     if (hints.isDensityQuery) {
-      val is: Map[String, String] = KryoLazyDensityCoprocessor.configure(sft, ranges, hbaseFilters, hints)
+      val rowRanges = Lists.newArrayList[RowRange]()
```

Comment:
Yes, this also is my intent.  The idea is to pass the query ranges to a MRRF and add that to a FilterList.

The FilterList and Scan are passed to the Coprocessor configure.